visualization/package.xml

```diff
@@ -31,6 +31,7 @@
   <build_depend>moveit_ros_perception</build_depend>
   <build_depend>cmake_modules</build_depend>
   <build_depend>rospy</build_depend>
+  <build_depend>rostest</build_depend>
```

Comment:
Using `test_depend` is even better http://answers.ros.org/question/115526/is-test_depend-actually-used-for-anything/